### ISSUE TITLE
[3.3.5] Ranged weps unable to attack at range

### ISSUE THREAD

**Issue by serfma** [TrinityCore/TrinityCore#13620]
These items are broken and are unable to be fired at any range:

Savage Gladiator's Longbow (42445)
Savage Gladiator's Heavy Crossbow (42446)
Savage Gladiator's Rifle (42447)
Hateful Gladiator's Longbow (42489)
Hateful Gladiator's Heavy Crossbow (42494)
Hateful Gladiator's Rifle (42484)

Deadly, Furious, Relentless, and Wrathful Gladiator's Bows/Crossbows/Rifles all work as intended except for the above list.

Edit: Found the fix for the above weapons. Under item_template for all 6 entries they were missing "ammo_type" and "RangedModRange". Both were set to 0, but once setting ammo_type to 2 or 3 and setting RangedModRange to 100 they all started working as intended. Spelltrigger_1 was set to 0, but other bows/ranged weps are set to 1, so I didn't touch it.

Core revision: e778ee14163a 2014-11-21 15:46:26 -0200 (3.3.5 branch)
Database: TDB 335.57

**Comment by tkrokli** [comment]
http://collab.kpsn.org/display/tc/item_template#item_template-ammo_type
```
The type of ammunition the item uses. Arrows = 2; Bullets = 3
```
http://collab.kpsn.org/display/tc/item_template#item_template-RangedModRange ;
`Range Modifier for bows/guns/crossbows: Default range is somewhere between 0.3 and 0.4 yards,
all blizzard ranged weapons have RangedModRange 100`

The information above gives us this neat update SQL for `item_template` :
```sql
-- Savage Gladiator's Longbow & Heavy Crossbow, Hateful Gladiator's Longbow & Heavy Crossbow:
UPDATE `item_template` SET `ammo_type`= 2, `RangedModRange` = 100 WHERE `entry` IN (42445,42446,42489,42494);
-- Savage Gladiator's Rifle (42447) & Hateful Gladiator's Rifle (42484) :
UPDATE `item_template` SET `ammo_type`= 3, `RangedModRange` = 100 WHERE `entry` IN (42447,42484);

**Comment by tkrokli** [comment]
http://www.wowhead.com/item=42445/savage-gladiators-longbow#comments :
Wowhead tag: `This item is not available to players.`
[[irelance on 2009/01/03 (Patch 3.0.3)]](http://www.wowhead.com/item=42445/savage-gladiators-longbow#comments:id=438279:reply=69884)
"He was trying to say that he was looking to spend all of his unused level 70 arena points on this weapon so that he would have a good bow to use at level 80. Too bad this weapon doesn't exist = / "

http://www.wowhead.com/item=42446/savage-gladiators-heavy-crossbow
Wowhead tag: `This item is not available to players.` (No comments in the comment section.)

http://www.wowhead.com/item=42447/savage-gladiators-rifle#comments
Wowhead tag: `This item is not available to players.`
None of the commenters have confirmed the item's existence on Retail.

http://www.wowhead.com/item=42489/hateful-gladiators-longbow#comments
Wowhead tag: `This item's source is no longer available/removed.`
Most of the comments seem to confirm that the item is not available in the game.

http://www.wowhead.com/item=42494/hateful-gladiators-heavy-crossbow#comments
Wowhead tag: `This item's source is no longer available/removed.`
No confirming comments about the item's existence on retail.

http://www.wowhead.com/item=42484/hateful-gladiators-rifle#comments
Wowhead tag: `This item's source is no longer available/removed.`
No confirming comments about the item's existence on retail.

Based on this information and no confirming information anywhere else, I vote for closing this issue.

**Comment by MrSmite** [comment]
Well there is this: http://eu.battle.net/wow/en/item/42445

Doesn't say how it's obtained but it indicates "BOP" and disenchanting levels. I think this is a TBC PvP item but I'm not 100% since I rarely did PvP.

**Comment by Aokromes** [comment]
I said that 1 year ago :P

**Comment by StormBytePP** [comment]
Maybe not, there are some items in ptr that don't have any loot nor obtain methods, but not deleted from db

**Comment by Rushor** [comment]
never used in game http://www.wowhead.com/item=42447/savage-gladiators-rifle#comments:id=1088002 a three people already said now

question: should we fix this stuff if i was no longer available in 3.3.5?

**Comment by Rushor** [comment]
jackpoz wrote:
> This issue is in the category "won't fix". It's just so cosmetic, so little priority that we can't even classify it as an issue.

so we will trade this issue like the one from https://github.com/TrinityCore/TrinityCore/issues/12918

The stuff was long longer available in the patch we are supporting (stuff was from 3.1.x ptr and we are only supporting 3.3.5)
